### PR TITLE
add nb filterd elements displayed in grid footer

### PR DESCRIPTION
**Pull request by jvitus**
part 1 b ) display nb row displayed after filter

https://www.pivotaltracker.com/story/show/137588133

### CODE REVIEW

File: Front/app/ns_modules/ns_grid/grid.view.js [hunk 1]
```diff
@@ -96,6 +98,20 @@ define([
         onAfterFilterChanged: function(){
           _this.handleSelectAllChkBhv();
           _this.clientSideFilter();
+
+          var tmp = _this.gridOptions.api.getFilterModel();
+          var p = 0;
+          for (var i in tmp) {
```
Comment (Matheo13) [inline Front/app/ns_modules/ns_grid/grid.view.js]: Object.keys(obj).length === 0
if ($.isEmptyObject(obj))
or _.isEmpty({});

**Comment by Matheo13** [comment]
Good bhv.

**Comment by Matheo13** [comment]
Ok for me I let you merge if you are ok as well.

**Review by Matheo13 — APPROVED** [review]

**Comment by jvitus** [comment]
Ok for me :)

**Pull request merged**